### ISSUE TITLE
Introduce dotenv files

### ISSUE THREAD

**Issue by connorshea** [salesagility/SuiteCRM#7620]
#### Issue
This is part of #7344, but I'm splitting it out into a separate issue.

Introduce `.env` files for use with testing, either [phpdotenv](https://github.com/vlucas/phpdotenv) or [Symfony Dotenv](https://github.com/symfony/dotenv).

This is mostly useful for running multiple CRMs side-by-side on the same computer, and also because it makes it easier to figure out what the local environment variables are.

For example, I have a CRM for contributing to the upstream and a CRM for our customized version of the CRM. I'd like to have a dotenv file in each so that I can set environment variables for the test suite that differ between the two sites.

CRM 1 is at http://localhost and CRM 2 is at http://localhost:8080, but the environment variable I have set on my Mac is set to `INSTANCE_URL=http://localhost`, which means that when I run the tests on CRM 2, I end up running them on the CRM 1 site. The only way to fix this is to change the environment variables every time I switch between sites, which is _really_ annoying.

If we introduced dotenv files, we could have the variables set in the root of the site, e.g. `CRM1/.env` and `CRM2/.env`.

CRM1 `.env`:
```
INSTANCE_URL="https://localhost"
DATABASE_NAME="automated_tests"
```

CRM2 `.env`: 
```
INSTANCE_URL="https://localhost:8080"
DATABASE_NAME="automated_tests2"
```

We could also have `.env.test` to separate environment variables between 'development' and 'test', e.g. when running the tests we don't want to mess with our database we use for development.

TL;DR the main benefits are:

- Easier to configure the environment variables for a site
- Make it possible to switch between two separate instances to run tests
- Everything is configured in the instance's directory, rather than some configuration being externally stored as environment variables.
